buildroot/share/scripts/MarlinBinaryProtocol.py

```diff
@@ -0,0 +1,434 @@
+import serial
```

Comment:
Was this file copied from elsewhere? If so, we probably need an attribution of some sort added to the top.

Reply:
> Was this file copied from elsewhere? If so, we probably need an attribution of some sort added to the top.

Elsewhere from the aforementioned PR #14817?
I don't know, but it looks like an implementation from user @p3p